src/js/events/initModal.js

```diff
@@ -10,90 +11,40 @@ const setTime = 2000;
 export async function initModal() {
   await delay(setTime);
   renderLoginModal();
-  mainLogin.addEventListener('mouseenter', handleModal);
-  mainLogin.addEventListener('mouseleave', removeModal);
-  mainShippingAddress.addEventListener('mouseenter', handleModal);
-  mainShippingAddress.addEventListener('mouseleave', removeModal);
+  mainLogin.addEventListener('mouseenter', e => {
+    modalRenderer('.modal', TEMPLATE.expandedLogin)();
+  });
+  mainLogin.addEventListener('mouseleave', e => {
+    modalRemover('.modal')(e);
+  });
```

Comment:
```suggestion
   mainLogin.addEventListener('mouseleave',  modalRemover('.modal'));
  });
```

modalRemover가 e를 받는 함수를 반환하기 때문에, 직접 e를 넣으면서 호출하지 않고 요렇게 전달할 수 있습니다. 
제 저번주 리뷰의 의도는 이게 맞는데 

>내부의 함수가 상위 함수의 어떤 변수나 상태를? 사용하는 것이 없어서

요런 리뷰를 받으셨다니 음.. 전달한 선택자 문자열을 리턴된 함수가 사용하는 것인데 사용하는게 없다는 것은 이해가 안 가네용..

아무튼 작성하신 방향은 제 저번주 리뷰 의도와 일치합니다!